src/test/java/Package/HomePage.java

```diff
@@ -0,0 +1,41 @@
+package Package;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class HomePage extends BasePage{
+
+    public HomePage (WebDriver givenDriver){
+        super (givenDriver);
+    }
+
+    By userAvatarIcon = By.cssSelector("img.avatar");
+    By nextSongIcon = By.cssSelector("//div/i[last()]");
+
+    By playSongIcon = By.xpath("//*[@data-testid=\"play-btn\"]");
+
+    By soundBar = By.xpath("//*[@id='mainFooter']");
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```